autoload/gundo.py

```diff
@@ -461,7 +461,6 @@ def make_nodes(self):
         # if the current seq_last and file are different, compute the new
         # values:
         if self.seq_last != seq_last:
-            vim.command('let s:has_supported_python = 0')
```

Comment:
@dsummersl Please let me know if deletion of this line causes any unexpected side effect